lib/src/css_mirroring/cssjanus_runner.dart

```diff
@@ -0,0 +1,30 @@
+library scissors.src.css_mirroring.cssjanus_runner;
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import '../utils/io_utils.dart';
+
+/// Executes cssjanus(https://github.com/cegov/wiki/tree/master/maintenance/cssjanus).
+/// Input: [source] css and [cssJanusPath] which points to an executable.
+/// Pipes in the source css to cssjanus.
+/// Output: css flipped from ltr to rtl orientation and vice-versa.
+Future<String> runCssJanus(String source, String cssJanusPath) async {
```

Comment:
I'm still uneasy with source: `css` seems to describe better what this string is (and sourceFlipped below, which should be flippedSource, would then become flippedCss)